examples/magento-graphcms/pages/p/[url].tsx

```diff
@@ -66,109 +68,110 @@ function ProductPage(props: Props) {
     relatedUpsells?.items?.find((item) => item?.uid === products?.items?.[0]?.uid),
   )
 
+  const cartButtonRef = useRef<HTMLButtonElement | null>(null)
+
```

Comment:
Ultimately we don't want to create a ref like this on the product page. It would be nice if we could create a plugin on de `AddProductToCartButton` and move the sticky-add-to-cart logic to that plugin. We could portal that component a little bit higher in the DOM, to prevent weird styling issues https://react.dev/reference/react-dom/createPortal